native/src/Navigator.tsx

```diff
@@ -178,141 +178,37 @@ const Navigator = (props: PropsType): ReactElement | null => {
-        options={{
-          header: () => null
-        }}
-      />
-      <Stack.Screen
-        name={DASHBOARD_ROUTE}
-        component={DashboardContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={CATEGORIES_ROUTE}
-        component={CategoriesContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={OFFERS_ROUTE}
-        component={OffersContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={SPRUNGBRETT_OFFER_ROUTE}
-        component={SprungbrettOfferContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={EXTERNAL_OFFER_ROUTE}
-        component={ExternalOfferContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={POIS_ROUTE}
-        component={PoisContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={EVENTS_ROUTE}
-        component={EventsContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={NEWS_ROUTE}
-        component={NewsContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={DISCLAIMER_ROUTE}
-        component={DisclaimerContainer}
-        options={{
-          header: defaultHeader
-        }}
-      />
-      <Stack.Screen
-        name={SEARCH_ROUTE}
-        component={SearchModalContainer}
-        options={{
-          header: () => null
-        }}
-      />
-      <Stack.Screen
-        name={PDF_VIEW_MODAL_ROUTE}
-        component={PDFViewModal}
-        options={{
-          header: transparentHeader
-        }}
-      />
-      <Stack.Screen
-        name={CHANGE_LANGUAGE_MODAL_ROUTE}
-        component={ChangeLanguageModalContainer}
-        options={{
-          header: transparentHeader
-        }}
-      />
-      <Stack.Screen
-        name={IMAGE_VIEW_MODAL_ROUTE}
-        component={ImageViewModal}
-        options={{
-          header: transparentHeader
-        }}
-      />
-      <Stack.Screen
-        name={FEEDBACK_MODAL_ROUTE}
-        component={FeedbackModalContainer}
-        options={{
-          header: transparentHeader
-        }}
-      />
-      <Stack.Screen
-        name={SETTINGS_ROUTE}
-        component={SettingsContainer}
-        options={{
-          header: settingsHeader
-        }}
-      />
-      <Stack.Screen
-        name={JPAL_TRACKING_ROUTE}
-        component={JpalTracking}
-        options={{
-          header: transparentHeader
-        }}
-      />
+    <Stack.Navigator initialRouteName={initialRoute.name} screenOptions={{ ...transitionPreset, headerMode: 'screen' }}>
+      <Stack.Group screenOptions={{ header: () => null }}>
```

Comment:
These `Group`components just set `options` for all child screens which looks a lot nicer imo.